web-components/src/components/tabs/scss/tab.scss

```diff
@@ -6,14 +6,14 @@
   box-shadow: none !important;
   outline: none;
 
-  button {
+  .tab-content {
     outline: $tab-main-focus-border;
```

Comment:
the inner element is now a div instead of a button so css needed updating.

In general it is better to style by class rather than type so this kind of change isn't necessary